scripts/components/OpenSearch/build.sh

```diff
@@ -77,19 +81,54 @@ cp -r ./build/local-test-repo/org/opensearch "${OUTPUT}"/maven/org
 
 [ -z "$PLATFORM" ] && PLATFORM=$(uname -s | awk '{print tolower($0)}')
 [ -z "$ARCHITECTURE" ] && ARCHITECTURE=`uname -m`
+[ -z "$DISTRIBUTION" ] && DISTRIBUTION="None"
```

Comment:
Default it to `tar` here, not another keyword `None`.

Reply:
fixed.